020-quarkus-http-non-application-endpoints/src/test/java/io/quarkus/qe/non_application/endpoint/CommonNonAppEndpoint.java

```diff
@@ -0,0 +1,53 @@
+package io.quarkus.qe.non_application.endpoint;
+
+import static io.restassured.RestAssured.given;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+
+public abstract class CommonNonAppEndpoint implements QuarkusTestResourceLifecycleManager {
```

Comment:
This class does not need to implement QuarkusTestResourceLifecycleManager. Also, the methods start and stop are not used.